bridges/scala-native-0.4/src/test/scala/bloop/scalanative/ScalaNativeToolchainSpec.scala

```diff
@@ -0,0 +1,90 @@
+package bloop.scalanative
+
+import bloop.cli.{Commands, OptimizerConfig}
+import bloop.engine.Run
+import bloop.logging.RecordingLogger
+
+import scala.concurrent.duration.Duration
+import java.util.concurrent.TimeUnit
+
+import bloop.data.{Platform, Project}
+import bloop.engine.tasks.toolchains.{ScalaJsToolchain, ScalaNativeToolchain}
+import bloop.util.TestUtil
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.experimental.categories.Category
+
+@Category(Array(classOf[bloop.FastTests]))
+class ScalaNativeToolchainSpec {
```

Comment:
Would it be possible to change the project build definition so that we reuse the tests from the native bridge 0.3? Given that the tests are the same, we would avoid copying the toolchain spec.